netmiko/cloudgenix/cloudgenix_ion.py

```diff
@@ -1,49 +1,58 @@
+import io
+from typing import Any, Union, Iterable
+
 from netmiko.cisco_base_connection import CiscoSSHConnection
 
 
 class CloudGenixIonSSH(CiscoSSHConnection):
-    def establish_connection(self):
-        super().establish_connection(width=100, height=1000)
+    def establish_connection(self) -> None:
+        self.channel.establish_connection(width=100, height=1000)
```

Comment:
So this is "fixed" but I don't think this code will ever be called.  If we need to ensure width/height are set to 100, what's the right hook to use?  Do we need a custom channel?

Reply:
Okay, thanks...yeah, I will have to figure out how to handle it. I am not sure if it is even needed any longer. 

I have created a Netmiko issue to track the problem.